.github/workflows/continuous-integration.yml

```diff
@@ -27,24 +30,31 @@ jobs:
       run: dotnet restore
 
     - name: Build
-      run: dotnet build --configuration WindowsRelease
+      run: |
+        dotnet build src/windows/Installer.Windows/Installer.Windows.csproj `
+         --configuration=Release `
+         --runtime=${{ matrix.runtime }}

```

Comment:
Interestingly, `dotnet build` [doesn't support](https://github.com/dotnet/sdk/issues/14281) setting `--runtime` at the solution level. I basically took the same logic as https://github.com/git-ecosystem/git-credential-manager/pull/1633 by setting it at the project level: the `Installer.Windows` project has all the necessary dependencies so this builds correctly.